nemo/core/classes/exportable.py

```diff
@@ -123,11 +123,16 @@ def export(
             if len(dynamic_axes) == 0:
                 dynamic_axes = None
 
+            if hasattr(self, 'symbolic'):
+                subj = self.symbolic
+            else:
+                subj = self.forward
```

Comment:
I guess this part will have to change? Should I keep it as symbolic instead of infer?